extensions/amp-sidebar/amp-sidebar.md

```diff
@@ -122,6 +122,10 @@ Specifies the display layout of the sidebar, which must be `nodisplay`.
 
 This attribute is present when the sidebar is open.
 
+##### data-close-button-aria-label**
+
+Optional string used as ARIA label for close button.
```

Comment:
This should be more like
"Optional attribute used to set ARIA label for the close button added for accessibility"

CC @bpaduch - to review the string